pkg/log/hclog.go

```diff
@@ -0,0 +1,58 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc/metadata"
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"kusionstack.io/kusion-module-framework/pkg/util/kfile"
+)
+
+const (
+	KusionModuleName = "KUSION_MODULE_NAME"
+	KusionTraceID    = "KUSION_TRACE_ID"
+)
+
+func GetModuleLogger(ctx context.Context) hclog.Logger {
+	traceID := getTraceID(ctx)
+	moduleName := getModuleName(ctx)
+	if moduleName == "" {
+		moduleName = "default"
```

Comment:
It might be better to use `kusionstack-default-module` rather than just `default` here, since we've had a lot of instances named `default` (backend, workspace, etc.) 😂

Reply:
fixed